src/SFA.DAS.CommitmentPayments.WebJob.UnitTests/SFA.DAS.CommitmentPayments.WebJob.UnitTests.csproj

```diff
@@ -89,7 +89,9 @@
     <Compile Include="Updater\WhenRunningUpdate.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="app.config" />
+    <None Include="app.config">
+      <SubType>Designer</SubType>
```

Comment:
Is this change required? Reading this article: https://social.msdn.microsoft.com/Forums/en-US/1c014433-95e1-46ba-8a8c-5dceaabb7ad3/what-is-subtypedesignersubtype-for-in-csproj?forum=csharpide, there is a bit of confusion about the IDE automatically adding and removing it under certain circumstances